climpred/graphics.py

```diff
@@ -230,3 +236,194 @@ def plot_bootstrapped_skill_over_leadyear(bootstrapped, plot_persistence=True, a
     ax.legend(frameon=False)
     ax.set_xlabel('Lead time [years]')
     return ax
+
+
+def check_only_climpred_dims(pe):
+    additional_dims = set(pe._datasets['initialized'].dims) - set(CLIMPRED_DIMS)
+    if len(additional_dims) != 0:
+        raise DimensionError(
+            f'{type(pe)}.plot() does not allow dimensions other '
+            f'than {CLIMPRED_DIMS}, found {additional_dims}. '
+            f'Please use .mean({additional_dims}) '
+            f'or .isel() before plot.'
+        )
+
+
+def plot_lead_timeseries_hindcast(
+    he, variable=None, ax=None, show_members=False, cmap='jet'
+):
+    """Plot datasets from HindcastEnsemble.
+
+    Args:
+        he (PredictionEnsemble): PredictionEnsemble.
```

Comment:
Should this be described as HindcastEnsemble since that's the only thing going through this function?